render-ws-java-client/src/main/java/org/janelia/render/client/newsolver/setup/RenderSetup.java

```diff
@@ -154,7 +154,8 @@ private static RenderSetup setupSolve(
 		// a HashMap where int is the z section, and string is the description (problem, restart, ...)
 		runParams.zToGroupIdMap = new HashMap<>();
 		for (final String groupId : Arrays.asList("restart", "problem")) { // NOTE: "problem" groupId is for future use
-			LOG.debug( "Querying: " + groupId );
+			if (LOG.isDebugEnabled())
+				LOG.debug("Querying: {}", groupId);
```

Comment:
There is no real need for the isDebugEnabled check here.  Making the log message parameterized is sufficient.  I only add the explicit `is<>Enabled` check when building the log message is expensive.

Reply:
Thanks for the tip! I'll exclude all such checks where building the log message doesn't do expensive function calls (i.e., all I suppose).